### PR TITLE
Purge modules

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes/no
| Did you apply meaningful labels to the pull request? | yes/no
| Did you perform a self review first? | yes/no
| Documentation reflects changes? | yes/no
| `CHANGELOG` reflects changes? | yes/no
| Unit/Functional tests reflect changes? | yes/no
| Did you perform browser testing? | yes/no
| Risk level | Low, Medium, High
| Relevant links | JIRA issue, bug reports, security alerts, etc.

### CODE REVIEW

File: ecms_base/config/install/purge.logger_channels.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+channels:
```
Comment (pfrilling) [inline ecms_base/config/install/purge.logger_channels.yml:1]: Should this maybe live in the `ecms_acquia` profile instead?

File: ecms_base/ecms_base.info.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ core_version_requirement: ^9
 # Modules that sub-profiles need must be listed here and_not in install.
 dependencies:
   - allowed_formats
+  - acquia_purge
```
Comment (pfrilling) [inline ecms_base/ecms_base.info.yml:10]: Should these be dependencies of the `ecms_acquia` profile?

File: ecms_base/ecms_base.install [hunk 3]
```diff
@@ -448,6 +448,19 @@ function ecms_base_update_9029(array &$sandbox): void {
  */
 function ecms_base_update_9030(array &$sandbox): void {
 
+  // Install the newly introduced purge modules.
+  $modules_to_install = [
```
Comment (pfrilling) [inline ecms_base/ecms_base.install:452]: Might need a rebase? I added the `ecms_publications` to the module install list too.

**Review by pfrilling — CHANGES_REQUESTED** [review]
A few changes/questions.

File: ecms_base/ecms_base.install (hunk 3 above)
Comment (ben-hamelin) [reply ecms_base/ecms_base.install:452]: Yep, I think everything's in place now

**Comment by ben-hamelin** [comment]
Ok feedback addressed

**Review by pfrilling — APPROVED** [review]
I think this looks good.

**Pull request merged**